ci-operator/jobs/janus-idp/backstage-showcase/janus-idp-backstage-showcase-main-periodics.yaml

```diff
@@ -231,28 +231,15 @@ periodics:
   labels:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-janus-idp-backstage-showcase-main-e2e-tests-nightly-ocp-v4-14
-  reporter_config:
```

Comment:
@josephca, for 4-15 you are missing the `reporter_config` for slack alerts.

Reply:
Thanks. Updated.